.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+# Trac ticket number
+<!-- Replace [number] with the corresponding Track ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
```

Comment:
```suggestion
<!-- Replace [number] with the corresponding Trac ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
```

Reply:
Great catch @LilyFoote, thank you! Fixing.